### PR TITLE
feat: migrate `Move chart to dashboard` modal

### PR DESCRIPTION
**Pull request by yasmin-e**
Closes: #5155 

### Description:
Technical changes:
```
- Migrated form to Mantine use-form
- Migrated all components to Mantine
- Added grouping by Space to Select dash dropdown
```
Screenshot 2023-06-21 at 14 29 52
Screenshot 2023-06-21 at 14 30 07
Screenshot 2023-06-21 at 14 30 14
Screenshot 2023-06-21 at 14 30 18
Screenshot 2023-06-21 at 14 30 25

### CODE REVIEW

File: packages/frontend/src/components/UserSettings/UserManagementPanel/index.tsx [hunk 1]
```diff
@@ -190,7 +190,7 @@ const UserListItem: FC<{
                                         <Title order={4}>Delete user</Title>
                                     </Group>
                                 }
-                                yOffset="30vh"
+                                centered
```
Comment (IrakliJani) [inline packages/frontend/src/components/UserSettings/UserManagementPanel/index.tsx:193]: is this part of the PR? its ok if its not an accidental change

File: packages/frontend/src/components/SavedDashboards/AddTilesToDashboardModal.tsx [hunk 2]
```diff
@@ -184,163 +182,149 @@ const AddTilesToDashboardModal: FC<AddTilesToDashboardModalProps> = ({
...
     return (
-        <Dialog
-            isOpen={isOpen}
-            onClose={onClose}
-            lazy
-            icon="control"
-            title="Add chart to dashboard"
+        <Modal
+            opened={isOpen}
+            onClose={() => onClose}
```
Comment (IrakliJani) [inline packages/frontend/src/components/SavedDashboards/AddTilesToDashboardModal.tsx:187]: should be `onClose()`

**Review by IrakliJani — APPROVED** [review]
looks good,  `onClose` needs to be fixed before merging

File: packages/frontend/src/components/UserSettings/UserManagementPanel/index.tsx (hunk 1 above)
Comment (yasmin-e) [reply packages/frontend/src/components/UserSettings/UserManagementPanel/index.tsx:193]: it's part of the PR, I was prev using yOffset cause I missed this prop so I fixed it there too for consistency

File: packages/frontend/src/components/SavedDashboards/AddTilesToDashboardModal.tsx (hunk 2 above)
Comment (yasmin-e) [reply packages/frontend/src/components/SavedDashboards/AddTilesToDashboardModal.tsx:187]: throws an error 🤔

**Comment by yasmin-e** [comment]
> Screenshot 2023-06-21 at 17 34 06
> Looks good! I've just noticed that you can't close the modal. You should be able to (like in the current version)

Added! ✅ 
Screenshot 2023-06-22 at 12 09 04

**Review by PriPatel — APPROVED** [review]

**Pull request merged**